lib/android-tools-info.ts

```diff
@@ -177,6 +177,24 @@ export class AndroidToolsInfo implements IAndroidToolsInfo {
 		}).future<boolean>()();
 	}
 
+	public getPathToAdbFromAndroidHome(): IFuture<string> {
+		return (() => {
+			if(this.androidHome) {
+				let pathToAdb = path.join(this.androidHome, "platform-tools", "adb") + (this.$hostInfo.isWindows ? ".exe" : "");
```

Comment:
yep, I do not need this, I'll remove it (I've added it for debugging purposes and I forgot to remove it).